languages/swift/exercises/concept/strings-and-characters/.docs/introduction.md

```diff
@@ -0,0 +1,111 @@
+Strings in Swift are a collection of characters, where characters are, more or less, a single printable element. So strings in Swift are able to contain Unicode characters and emoji.
+
+Strings are easily created in Swift through string literals. String literals are similar to those seen in many other programming languages, characters enclosed between a pair of double quotation marks (`"`). String literals can include the following special characters:
+\\0 (null character), \\\\ (backslash), \\t (horizontal tab), \\n (newline), \\r (carriage return), \\" (double quotation mark) and \\' (single quotation mark).
+
+An empty string is represented by two double quotation marks with nothing between them.
+
+```swift
+let hello = "Hello, World!"
+let sausage = "Weißwurst is a tasty sausage."
+var fam = "This is my family: 👨‍👩‍👦‍👦, this is our dog: 🐶"
+var empty = ""
+```
+
+Character literals in Swift look just like string literals, however, only one character is allowed between the quotation marks. Since the two literals look the same, Swift will default to inferring that characters are strings unless the type is specified by an explicit annotation or the context can tell the compiler otherwise.
+
+```swift
+let aString = "A"
+// Swift infers this to be a String
+
+var aChar: Character = "A"
+// Swift now knows to make this a Character
+
+let badChar: Character = "Too many characters"
+// Error: Cannot convert value of type 'String' to specified type 'Character'
+```
+
+Like other constructs in Swift, characters and strings are immutable if defined using `let` and mutable if defined using `var`. So in the above examples, `hello` and `sausage` are immutable strings while `fam` and `empty`, AND `aChar` are mutable.
```

Comment:
Is this line needed? It sounds like everything works just like the students would expect after having done the `basics` exercise, so I feel that this probably does not add any value here (you could include it in the `after.md` document if you want to emphasize this fact).